ports/luafilesystem/CMakeLists.txt

```
@@ -1,17 +1,43 @@
-cmake_minimum_required(VERSION 3.0)
-project(luafilesystem)
-
-find_path(LUA_INCLUDE_DIR lua.h PATH_SUFFIXES lua)
-find_library(LUA_LIBRARY lua)
-set(LFS_INCLUDES ${LUA_INCLUDE_DIR})
-set(LFS_LIBRARIES ${LUA_LIBRARY})
-
-add_library( lfs src/lfs.h src/lfs.c src/lfs.def)
-
-target_include_directories(lfs PRIVATE ${LFS_INCLUDES})
-target_link_libraries(lfs PRIVATE ${LFS_LIBRARIES})
-
-install(TARGETS lfs
-    RUNTIME DESTINATION bin
-    LIBRARY DESTINATION lib
-    ARCHIVE DESTINATION lib)
+cmake_minimum_required(VERSION 3.11)
+project(luafilesystem)
+
+find_path(LUA_INCLUDE_DIR lua.h PATH_SUFFIXES lua)
+find_library(LUA_LIBRARY lua)
+set(LFS_INCLUDES ${LUA_INCLUDE_DIR})
+set(LFS_LIBRARIES ${LUA_LIBRARY})
+
+add_library(lfs src/lfs.h src/lfs.c src/lfs.def)
+
+target_include_directories(lfs PRIVATE ${LFS_INCLUDES})
+target_link_libraries(lfs PRIVATE ${LFS_LIBRARIES})
+
+install(TARGETS lfs
+    EXPORT "unofficial-lfs-targets"
```

I would suggest to align the targets file with the config name (which must be based on the port name):
```suggestion
    EXPORT "unofficial-${PROJECT_NAME}-targets"
```